src/main/java/samza/examples/sql/samza-sql-samples-yarn/src/main/sql/samza.sql

```diff
@@ -0,0 +1,3 @@
+insert into log.outputStream
```

Comment:
I'm not sure if this example is required. We can delete it.

Reply:
removed